src/App.css

```
@@ -0,0 +1,229 @@
+body {
+   background-color: #cacccd65;
+   color:#121212;
+   margin:0 0 0 0;
```

можно просто 0
```suggestion
   margin: 0;
```
еще можно использовать 2 числа - первое для верх и них, а второе для лево и право